meta/documents/changelog_de.md

```diff
@@ -1,5 +1,16 @@
 # Release Notes für Feedback
 
+## v4.0.7 (2022-10-21)
```

Comment:
Git compare link needed in public plugins

Reply:
done